### DISCUSSION TITLE
Today's Meeting starting time

### DISCUSSION THREAD

**Discussion by ACAwebbuilder** [code4recovery/12-step-meeting-list#1228] [Q&A]
Hello!

I am wanting to see if there is a way to tweak the timing of meetings shown with the link for today's meetings (?post_type=tsml_meeting).  We use this to show today's meetings, but it is only showing the upcoming meetings, so it doesn't show meetings that have already started.  Is there a way for it to show all meetings that started within the last hour as well as those upcoming?
I hope this makes sense.
Thank you!

**Comment by joshreisner** [comment]
it should be doing that -- see https://aasanjose.org/meetings as an example

there is a yellow "15 meetings in progress" banner right now.

are your meeting end times set? what's the link again?
